src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2957,9 +2955,9 @@ private function registerCachingHttpClient(array $options, array $defaultOptions
 
         $container
             ->register($name.'.caching', CachingHttpClient::class)
-            ->setDecoratedService($name, null, 13) // between RetryableHttpClient (10) and ThrottlingHttpClient (15)
```

Comment:
is it expected to lose this relation to RetryableHttpClient and ThrottlingHttpClient ?